starters/gatsby-starter-plugin/README.md

```diff
@@ -0,0 +1,111 @@
+<p align="center">
+  <a href="https://www.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Starter for a Gatsby Plugin
+</h1>
+
+A minimal boilerplate for the essential files Gatsby looks for in a plugin.
+
+## 🚀 Quick start
+
+To get started creating a new plugin, you can follow these steps:
+
+1. Initialize a new plugin from the starter with `gatsby new`
+
+```
+gatsby new my-plugin https://github.com/gatsbyjs/gatsby-starter-plugin
+```
+
+If you already have a Gatsby site, you can use it. Otherwise, you can [create a new Gatsby site](https://www.gatsbyjs.org/tutorial/part-zero/#create-a-gatsby-site) to use your plugin in with the `gatsby new` command again.
```

Comment:
```suggestion
If you already have a Gatsby site, you can use it. Otherwise, you can [create a new Gatsby site](https://www.gatsbyjs.org/tutorial/part-zero/#create-a-gatsby-site) to test your plugin.
```